### PR TITLE
Created UCSBOrganization API with 5 methods

### LINKED ISSUE #12: Create `UCSBOrganizationController`, add `GET` (index) and `POST` (create)
# Acceptance Criteria:

- [x] There is a controller file `UCSBOrganizationController.java`
  in the expected directory.
- [x] In `UCSBOrganizationController.java` there is 
  code for a `GET /api/UCSBOrganization/all` endpoint 
  that returns a JSON list of all `UCSBOrganization`s in the database.
  (We sometimes call this an *index* action since it lists all
  items in the database.)
- [x] In `UCSBOrganizationController.java` there is 
  code for a `POST /api/UCSBOrganization/post` endpoint
  that can be used to create a new entry in the table. (This
  is a *create* action.)
- [x] The Swagger-UI endpoints for these are well documented so that
  any member of the team can understand what they are for and
  how to use them.
- [x] The `POST` endpoint works as expected, in the sense that new
  records can be added to the database (on localhost).
- [x] The `GET` endpoint works as expected, in the sense that the new
  records that are added show up (on localhost).
- [x] The `GET` and `POST` endpoints work as expected when the 
  app is deployed to Dokku.
- [x] There is full test coverage (Jacoco) for the methods in 
  `UCSBOrganizationController.java`
- [x] There is full mutation test coverage (Pitest) for the methods in
  `UCSBOrganizationController.java`

### PR DESCRIPTION
**Pull request by yuanchenmeng**
# Overview
I created UCSBorganization API located at host/api/UCSBorganization with GET, DELETE, PUT, POST methods.
Basic CRUD operations work well on both dokku and localhost. It's deployed at team02-qa for review.

image

Closes #12, #13, #14, #15

### CODE REVIEW

**Comment by MPriston** [comment]
It looks like it is failing Pitest mutation, did you get all the cases for the test cases?

File: src/main/java/edu/ucsb/cs156/example/controllers/UCSBOrganizationController.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    @PutMapping("")
+    public UCSBOrganization updateHelpRequest(
+            @Parameter(name="orgCode") @RequestParam String orgCode,
+            @RequestBody @Valid UCSBOrganization incoming) {
+        log.info("incoming={}", incoming);
+        log.info("orgCode={}", orgCode);
+        UCSBOrganization ucsborg = uCSBOrganizationRepository.findById(orgCode)
+                .orElseThrow(() -> new EntityNotFoundException(UCSBOrganization.class, orgCode));
+
+        ucsborg.setOrgCode(incoming.getOrgCode());
```
Comment (MPriston) [inline src/main/java/edu/ucsb/cs156/example/controllers/UCSBOrganizationController.java]: Here is where your test is failling, you shouldn't update the code when doing the put, if you delete this line it should solve your pitest failling problem

**Review by MPriston — CHANGES_REQUESTED** [review]
If you delete this line it should go through

File: src/main/java/edu/ucsb/cs156/example/controllers/UCSBOrganizationController.java (hunk 1 above)
Comment (yuanchenmeng) [reply src/main/java/edu/ucsb/cs156/example/controllers/UCSBOrganizationController.java]: Thank you, I find it as well.

**Review by MPriston — APPROVED** [review]
LGTM

**Pull request merged**